iotfunctions/stages.py

```diff
@@ -784,21 +732,17 @@ def _get_sql_statement(self):
                             f"INSERT ({all_columns_list}) " \
                             f"VALUES (SOURCE.{', SOURCE.'.join(available_columns)})"
 
-            try:
+        try:
                 statement = ibm_db.prepare(self.dms.db_connection, raw_statement)
```

Comment:
Indentation is too big (Actually, it does not really matter here because is a one-line block but try to get used to correct indentation)